### DISCUSSION TITLE
Spelling error with perceptron

### DISCUSSION THREAD

**Discussion by digger-yu** [streetsidesoftware/cspell-dicts#2137] [Q&A]
![image](https://user-images.githubusercontent.com/55081697/234441466-e0199b10-a129-4876-a39a-26b7a43d9d0e.png)

https://en.wikipedia.org/wiki/Multilayer_perceptron

**Comment by Jason3S** [comment]
@digger-yu,

Please add `perceptron` to [dictionaries/software-terms/src/software-terms.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-terms.txt).

**Reply by digger-yu** [reply]
done.
